### PR TITLE
Modify for uadk epoll wait 

### PR DESCRIPTION
**Pull request by lin755**

### CODE REVIEW

**Comment by lin755** [comment]
It needs more review， so please **don't merge** now, thanks@hzhuang1

**Review by Liulongfang — COMMENTED** [review]
OK

File: drv/hisi_comp.c [hunk 1]
```diff
@@ -883,8 +886,6 @@ static int hisi_zip_comp_send(handle_t ctx, struct wd_comp_msg *msg, void *priv)
 		return ret;
 	}
 
-	hisi_qm_enable_interrupt(ctx, msg->is_polled);
-
```
Comment (hzhuang1) [inline drv/hisi_comp.c:887]: You didn't explain why to remove hisi_qm_enable_interrupt().

File: test/sanity_test.sh [hunk 2]
```diff
@@ -363,6 +364,8 @@ run_zip_test_v2()
 	sw_dfl_hw_ifl /tmp/syslog
 	hw_dfl_sw_ifl /tmp/syslog
 	hw_dfl_hw_ifl /tmp/syslog
+	export WD_COMP_EPOLL_EN=0
+	hw_dfl_hw_ifl /tmp/syslog
```
Comment (hzhuang1) [inline test/sanity_test.sh:368]: If you want to change the environment variable, it's better not to define it as global variable.
```
WD_COMP_EPOLL_EN=1 hw_dfl_hw_ifl /tmp/syslog
WD_COMP_EPOLL_EN=0 hw_dfl_hw_ifl /tmp/syslog
```

File: drv/hisi_comp.c (hunk 1 above)
Comment (lin755) [reply drv/hisi_comp.c:887]: add needed information to commit message now.

File: test/sanity_test.sh (hunk 2 above)
Comment (lin755) [reply test/sanity_test.sh]: It is modified now, please review again.

**Pull request merged**